### ISSUE TITLE
Feature request: sfdx-hardis.yml option to include hardis command in tooltip

### ISSUE THREAD

**Issue by readeral** [hardisgroupcom/vscode-sfdx-hardis#178]
Sometimes cross-referencing between [commands docs](https://sfdx-hardis.cloudity.com/commands/) and the extension commands is difficult. Optionally including the id as part of the tooltip output can help mentally connecting command button to CLI commands.

Something like this:

```
if (projectConfig.enhancedTooltips) {
      hardisCommands = this.enhancedTooltips(hardisCommands);
}

private async enhancedTooltips(hardisCommands: Array<any>) {
    hardisCommands = hardisCommands.map((item: any) => {
      if (item.id) {
        item.tooltip = item.tooltip + "\n" + item.id
      }
      return item
    })
   return hardisCommands
  }
```

**Comment by nvuillam** [comment]
@readeral did you try a right click on a command  ? :)

![image](https://github.com/user-attachments/assets/44064165-fa37-403d-bfb8-0a8b92bb0b74)

![image](https://github.com/user-attachments/assets/e0c4bbb7-cf85-4ed8-b7cd-c406827c7238)

![image](https://github.com/user-attachments/assets/ffeb9bd2-5dfe-4912-8b73-b10001c367b8)

**Comment by nvuillam** [comment]
@readeral solved in v5.4.0 :)

![image](https://github.com/user-attachments/assets/b428fa52-255a-4e4c-b098-cfb2577d8617)

**Comment by readeral** [comment]
Amazing, thanks! Sorry I didn't reply to this one, I've been in the weeds a little trying to ship some stuff! Definitely want to come back to some of my contribution ideas, just been a bit under the pump.
